### PR TITLE
https://issues.redhat.com/browse/ACM-7843 Update limitations

### PR DESCRIPTION
**Pull request by oafischer**
2.9 only.

### CODE REVIEW

File: release_notes/known_issues_application.adoc [hunk 1]
```diff
@@ -38,9 +38,28 @@ When you use the Argo CD pull model to deploy `ApplicationSet` applications and
...
+As a result, if the Argo CD application is propagated to the local cluster by the Argo CD pull model, the local cluster Argo CD application is not cleaned up, even though the local cluster is removed from the placement decision of the Argo CD `ApplicationSet`.
+
+To work around the issue and clean up the local cluster Argo CD application, remove the `skip-reconcile` annotation from the local cluster Argo CD application. See the following annotation:
+
+----
+annotations:
+    argocd.argoproj.io/skip-reconcile: "true"
+----
+
+Additionally, if you manually refresh the pull model Argo CD application in the console, the refresh is not processed and the refresh button in the console is disabled.
```
Comment (xiangjingli) [inline release_notes/known_issues_application.adoc]: if you manually refresh the pull model Argo CD application in the console
==========>
if you manually refresh the pull model Argo CD application in the ArgoCD application console

The console is ArgoCD application console, not the ACM application console

Comment (oafischer) [reply release_notes/known_issues_application.adoc]: Where can I find the Argo CD app console? I would like to take a look at it to better understand it. Can you get there from the OCP console or is it a separate thing?

Comment (swopebe) [reply release_notes/known_issues_application.adoc]: ish: https://argo-cd.readthedocs.io/en/stable/

Comment (xiangjingli) [reply release_notes/known_issues_application.adoc]: This is the separate console. Users need to install the openshift gitops operator on the ACM hub cluster to access the ArgoCD applications console. The openshift gitops operator is the downstream project of the ArgoCD. 

I have sent the demo link to @oafischer for his better understanding

Comment (oafischer) [reply release_notes/known_issues_application.adoc]: Thanks, updated!

**Review by xiangjingli — APPROVED** [review]

**Comment by dockerymick** [comment]
It seems like the updates were made. As a reminder, Oliver said he supports us merging the PR after receiving approval. Here is the screen capture
Screenshot 2023-11-27 at 1 56 45 PM

File: release_notes/known_issues_application.adoc [hunk 2]
```diff
@@ -38,9 +38,28 @@ When you use the Argo CD pull model to deploy `ApplicationSet` applications and
...
 //2.8.0: 3910
+//2.9:ACM-7843
 
 The hub cluster application set deploys to target managed clusters, but the local cluster, which is a managed hub cluster, is excluded as a target managed cluster.
 
+As a result, if the Argo CD application is propagated to the local cluster by the Argo CD pull model, the local cluster Argo CD application is not cleaned up, even though the local cluster is removed from the placement decision of the Argo CD `ApplicationSet`.
+
+To work around the issue and clean up the local cluster Argo CD application, remove the `skip-reconcile` annotation from the local cluster Argo CD application. See the following annotation:
+
+----
```
Comment (dockerymick) [inline release_notes/known_issues_application.adoc:50]:
```suggestion
[source,yaml]
----
```

File: release_notes/known_issues_application.adoc [hunk 3]
```diff
@@ -38,9 +38,28 @@ When you use the Argo CD pull model to deploy `ApplicationSet` applications and
...
+annotations:
+    argocd.argoproj.io/skip-reconcile: "true"
+----
+
+Additionally, if you manually refresh the pull model Argo CD application in the *Applications* section of the Argo CD console, the refresh is not processed and the *REFRESH* button in the Argo CD console is disabled.
+
+To work around the issue, remove the `refresh` annotation from the Argo CD application. See the following annotation:
+
+----
```
Comment (dockerymick) [inline release_notes/known_issues_application.adoc:60]:
```suggestion
[source,yaml]
----
```

File: release_notes/known_issues_application.adoc [hunk 4]
```diff
@@ -38,9 +38,28 @@ When you use the Argo CD pull model to deploy `ApplicationSet` applications and
 [#argo-pull-model-controller-local]
 == Local cluster is excluded as a managed cluster for pull model
 //2.8.0: 3910
+//2.9:ACM-7843
 
 The hub cluster application set deploys to target managed clusters, but the local cluster, which is a managed hub cluster, is excluded as a target managed cluster.
 
+As a result, if the Argo CD application is propagated to the local cluster by the Argo CD pull model, the local cluster Argo CD application is not cleaned up, even though the local cluster is removed from the placement decision of the Argo CD `ApplicationSet`.
```
Comment (dockerymick) [inline release_notes/known_issues_application.adoc]:
```suggestion
As a result, if the Argo CD application is propagated to the local cluster by the Argo CD pull model, the local cluster Argo CD application is not cleaned up, even though the local cluster is removed from the placement decision of the Argo CD `ApplicationSet` resource.
```

**Review by dockerymick — COMMENTED** [review]
@swopebe will you take a look at my suggestions? If you agree with them, please approve and merge this

**Review by swopebe — APPROVED** [review]

**Comment by dockerymick** [comment]
@oafischer thanks for starting on this PR and thanks @swopebe for the thumbs up and approval. I committed my suggestions. Merging this PR

**Review by dockerymick — APPROVED** [review]

**Pull request merged**